tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "noEmit": true,
+    "strict": true
+  },
+  "exclude": ["node_modules", "release"]
+}
```

Comment:
Minimal TypeScript project file with `strict` mode enabled for maximum compatibility.